scripts/publish-to-unity-package.sh

```diff
@@ -19,6 +19,19 @@ echo Moving files to Unity package...
 pushd bin/release/netstandard2.1/publish
 rm Newtonsoft.Json.dll
 rm UnityEngine.dll
+
+# Check if io.chainsafe.web3-unity.lootboxes directory exists
+if [ -d "../../../../../../Packages/io.chainsafe.web3-unity.lootboxes" ]; then
+    rm -rf ../../../../../../Packages/io.chainsafe.web3-unity.lootboxes/Runtime/Libraries
+    mkdir -p ../../../../../../Packages/io.chainsafe.web3-unity.lootboxes/Runtime/Libraries
+    cp Chainsafe.Gaming.Chainlink.dll ../../../../../../Packages/io.chainsafe.web3-unity.lootboxes/Runtime/Libraries
+    cp Chainsafe.Gaming.LootBoxes.Chainlink.dll ../../../../../../Packages/io.chainsafe.web3-unity.lootboxes/Runtime/Libraries
+fi
```

Comment:
I'm getting a compile error after running this script, `Multiple precompiled assemblies with the same name Chainsafe.Gaming.Chainlink.dll included on the current platform. Only one assembly with the same name is allowed per platform.`

There's two references of `Chainsafe.Gaming.Chainlink.dll` one in `Packages/io.chainsafe.web3-unity.lootboxes/Chainlink/Runtime/Libraries/Chainsafe.Gaming.Chainlink.dll` and another in `Packages/io.chainsafe.web3-unity.lootboxes/Runtime/Libraries/Chainsafe.Gaming.Chainlink.dll`